syncserver.ini

```diff
@@ -42,3 +43,16 @@ identity_provider = https://accounts.firefox.com/
 # you tell the app that it's on HTTPS but it's really on HTTP, so it should
 # only be used as a last resort and after careful checking of server config.
 force_wsgi_environ = false
+
+[tokenserver]
+# Use a custom MySQL based syncstorage node hosted at http://localhost:8000
+
+# node_url = http://localhost:8000
+# sqluri = pymysql://sample_user:sample_password@127.0.0.1/syncstorage_rs
+
+[endpoints]
+# Replace syncserver endpoints with alternate server implementation, ie:
+# MySQL based syncstorage-rs 1.5 server hosted at http://localhost:8000/1.5 for UID "1"
+
+# "{node}/1.5/{uid}"
+# sync-1.5 = "http://localhost:8000/1.5/1"
```

Comment:
I don't think we should be hard-coding the uid here, does it work if you use `{uid}` instead?

Reply:
ah, yup; lemme change that.

Reply:
(it was meant as an example, but can adjust so it's extra clear)